### ISSUE TITLE
Training scripts of Pissa with LLaMA-2-7b for 3 epoches

### ISSUE THREAD

**Issue by lucasliunju** [GraphPKU/PiSSA#18]
Hi,

I found the results on Figure 4(c) is about 3 epoches. May I ask the training scripts about this experiment or some important hyper-parameters, such as learning rate, rank, batch size?

**Comment by fxmeng** [comment]
The hyperparameters for the 3-epoch experiment are exactly the same as those provided in the project script. You just need to use all the training data and change the number of epochs to 3.
